### PR TITLE
98 Create organizational unit scene

### LINKED ISSUE #98: Add user and organization pages
For show + edit, delete.

### PR DESCRIPTION
**Pull request by phyrog**
Closes #98.

Also does these things:

- Refactors some things according to #260
- Change the way the main routes are rendered (`HeaderAndBody`)
- introduces [stampit](https://github.com/stampit-org/stampit) + Faker.js (sth. like FactoryBot)
- introduces [jest-plugin-set](https://github.com/negativetwelve/jest-plugins/blob/master/packages/jest-plugin-set/README.md), which makes testing more like RSpec (with lazily evaluated variables)
- makes testing easier by importing some things globally in tests (`React`, `Router`, `render`, `shallow`, `mount`, etc)

Note that this PR probably includes multiple (or no) codestyles, to match existing files. I'd suggest leaving the code-style changes to a (or multiple) next PR to fix things globally.

### CODE REVIEW

File: src/components/VersionWarning.js [hunk 1]
```diff
@@ -53,4 +56,16 @@ export class VersionWarning extends Component {
   }
 }
 
-export default VersionWarning;
+export const VersionWarningWithData = graphql(versionQuery, {
+  props: props => ({
```
Comment (eugenk) [inline src/components/VersionWarning.js]: This line is not covered by the tests. Should there be a comment to ignore it?

**Comment by eugenk** [comment]
When I sign in as ada
And I navigate to http://localhost:3001/seed-user-organization/settings/members
And I click on a Trash button to remove a member from an organization
And I click on "Cancel" in the modal
Then I see a white page
And the console displays errors:

```
[Error] Warning: setState(...): Expected the last optional `callback` argument to be a function. Instead received: [object Object].

Invariant Violation: Invalid argument passed as callback. Expected a function. Instead received: [object Object]
logCapturedError — react-dom.development.js:9747

The above error occurred in the <withState(PureMemberSettings)> component:
    in withState(PureMemberSettings) (created by Formik)
```

**Comment by eugenk** [comment]
When I am not signed in
And I navigate to http://localhost:3001/seed-user-organization/settings/members
Then I see a white page
And the console displays errors:
```
[Error] TypeError: null is not an object (evaluating '_ref$data$organizatio2.role')

The above error occurred in the <Header> component:
    in Header (at HeadAndBody.js:11
```

**Comment by eugenk** [comment]
When I am signed in as any user
And I sign out
Then I see a white page
And the console displays errors:
```
TypeError: null is not an object (evaluating '_ref$data$organizatio2.role')

The above error occurred in the <Header> component:
    in Header (at HeadAndBody.js:11)
```

**Comment by eugenk** [comment]
I think some of the pages are missing a `Redirect`.

**Comment by eugenk** [comment]
I reviewed the code and some of the components are missing a Redirect. However, I'm not entirely sure if that's the reason for the errors.

**Comment by phyrog** [comment]
Bugs should be fixed in 7c6f1c4, don't know why stickler has some kind of problem, but it probably has nothing to do with the code.

**Pull request merged**